### PR TITLE
Updates a number of entries to current status

### PR DESCRIPTION
**Pull request by DougReeder**
Also remove redundant Vert.x entry.

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -727,7 +727,7 @@
   category:
     - server
     - data-validation
-  language: TypeScript
+  language: Node.js + Koa, Express or Hapi + TypeScript
```
Comment (philsturgeon) [inline _data/tools.yml:730]:
```suggestion
  language: Node.js / TypeScript
```

File: _data/tools.yml [hunk 2]
```diff
@@ -738,7 +738,7 @@
 
 - name: Vert.x Web Api Contract
   category: server
-  language: Java, Kotlin, JavaScript, Groovy, Ruby, Ceylon & Scala
+  language: Vert.x + Java, Kotlin, JavaScript, Groovy, Ruby, Ceylon or Scala
```
Comment (philsturgeon) [inline _data/tools.yml:741]:
```suggestion
  language: Java, Kotlin, JavaScript, Groovy, Ruby, Ceylon or Scala
```

File: _data/tools.yml [hunk 3]
```diff
@@ -747,18 +747,18 @@
...
 
-- name: Vert.x Web API Service
+- name: express-openapi
   category: server
-  language: Java
-  link: https://vertx.io/docs/vertx-web-api-service/java/
-  github: https://github.com/vert-x3/vertx-web/
-  description: Create API service proxies using event bus with request/response validation
-  v2: false
+  language: Node.js + Express (optionally + Typescript)
```
Comment (philsturgeon) [inline _data/tools.yml:752]:
```suggestion
  language: Node.js / Typescript
```

File: _data/tools.yml [hunk 4]
```diff
@@ -1323,6 +1326,16 @@
     Supports inheritance with discriminators, Jackson annotations and custom interceptors.
   v3: true
 
+- name: fetch-openapi
```
Comment (philsturgeon) [inline _data/tools.yml:1329]: Seems to only support OAS2 which is against best practice now OAS3.1 is out.

File: _data/tools.yml [hunk 5]
```diff
@@ -1042,10 +1045,10 @@
     - data-validators
     - description-validators
   github: https://github.com/isa-group/oas-tools
-  language: Node.js
+  language: Node.js + Express
```
Comment (philsturgeon) [inline _data/tools.yml:1048]: Let's not put frameworks in here. NodeJS is kinda unfortunately a language as its a whole environment beyond just JS/TS but I dont want to list Express and all the other WAFs things work with or it'll get bloated.

Comment (DougReeder) [reply _data/tools.yml:1047]: Framework really ought to be **somewhere**. If you're using NestJS or Meteor, a tool for Express won't do you any good.

File: _data/tools.yml (hunk 4 above)
Comment (DougReeder) [reply _data/tools.yml:1328]: Yeah, that should be updated.

File: _data/tools.yml [hunk 6]
```diff
@@ -747,18 +747,18 @@
...
+  link: https://www.npmjs.com/package/express-openapi
+  github: https://github.com/kogosoftwarellc/open-api/tree/master/packages/express-openapi
+  description: An unopinionated OpenAPI framework for Express, which supports Promise based middleware, response handlers and Security Filtering.
+  v2: true
   v3: true
 
 - name: BaucisJS + baucis-openapi3
   category: server
-  language: JavaScript
+  language: Node.js + Express + Baucis
```
Comment (philsturgeon) [inline _data/tools.yml:761]:
```suggestion
  language: Node.js (Express)
```

Lets do this. Baucis is redundant as its in the name.

File: _data/tools.yml [hunk 7]
```diff
@@ -1042,10 +1043,10 @@
     - data-validators
     - description-validators
   github: https://github.com/isa-group/oas-tools
-  language: Node.js
+  language: Node.js + Express
```
Comment (philsturgeon) [inline _data/tools.yml:1046]:
```suggestion
  language: Node.js
```

File: _data/tools.yml [hunk 8]
```diff
@@ -1512,7 +1513,7 @@
 - name: openapi-processor-spring
   category:
     - server
-  language: Java
+  language: Java + Spring Boot
```
Comment (philsturgeon) [inline _data/tools.yml:1516]:
```suggestion
  language: Java
```

File: _data/tools.yml [hunk 9]
```diff
@@ -1593,11 +1594,11 @@
 
 - name: SpringFox
   category: server
-  language: Java, Kotlin, Groovy, Ruby
+  language: Java, Kotlin, Groovy, or Ruby + Spring Boot
```
Comment (philsturgeon) [inline _data/tools.yml:1597]:
```suggestion
  language: Java, Kotlin, Groovy, or Ruby
```

File: _data/tools.yml [hunk 10]
```diff
@@ -747,18 +746,18 @@
...
+  link: https://www.npmjs.com/package/express-openapi
+  github: https://github.com/kogosoftwarellc/open-api/tree/master/packages/express-openapi
+  description: An unopinionated OpenAPI framework for Express, which supports Promise based middleware, response handlers and Security Filtering.
+  v2: true
   v3: true
 
 - name: BaucisJS + baucis-openapi3
   category: server
-  language: JavaScript
+  language: Node.js (Express)
```
Comment (philsturgeon) [inline _data/tools.yml:760]:
```suggestion
  language: Node.js
```

File: _data/tools.yml [hunk 11]
```diff
@@ -769,7 +768,7 @@
 
 - name: "@smartrecruiters/openapi-first"
   category: server
-  language: Node.js
+  language: Node.js + Express
```
Comment (philsturgeon) [inline _data/tools.yml:771]:
```suggestion
  language: Node.js
```

File: _data/tools.yml [hunk 12]
```diff
@@ -789,7 +788,7 @@
 
 - name: OpenAPI Enforcer Middleware
   category: server
-  language: Node.js
+  language: Node.js + Express (or Connect?)
```
Comment (philsturgeon) [inline _data/tools.yml:791]:
```suggestion
  language: Node.js
```

File: _data/tools.yml [hunk 13]
```diff
@@ -806,11 +805,11 @@
   github: https://github.com/tomi-vanek/microts
   description: Take an OpenAPI description and generate TypeScript projects via Docker.
   v2: true
-  v3: true
+  v3: false
 
 - name: API Platform
   category: server
-  language: PHP
+  language: PHP (reccommended + Symfony)
```
Comment (philsturgeon) [inline _data/tools.yml:812]:
```suggestion
  language: PHP
```

File: _data/tools.yml [hunk 14]
```diff
@@ -853,7 +852,7 @@
 - name: "@eropple/nestjs-openapi3"
   category:
     - server
-  language: Node.js
+  language: Node.js + NestJS
```
Comment (philsturgeon) [inline _data/tools.yml:855]:
```suggestion
  language: Node.js
```

File: _data/tools.yml [hunk 15]
```diff
@@ -866,7 +865,7 @@
 - name: "@nestjs/swagger"
   category:
     - server
-  language: TypeScript
+  language: Node.js + NestJS + TypeScript
```
Comment (philsturgeon) [inline _data/tools.yml:868]:
```suggestion
  language: Node.js
```

File: _data/tools.yml [hunk 16]
```diff
@@ -880,7 +879,7 @@
   category:
     - server
     - mock
-  language: Python
+  language: Python + Django, Falcon or Flask
```
Comment (philsturgeon) [inline _data/tools.yml:882]:
```suggestion
  language: Python
```

**Review by philsturgeon — APPROVED** [review]

**Pull request merged**